code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -304,3 +304,26 @@
 	name = "emperor penguin"
 	desc = "Emperor of all he surveys."
 
+/mob/living/simple_animal/penguin/wizard
+
+/mob/living/simple_animal/penguin/wizard/verb/change_name()
```

Comment:
tbh, don't duplicate code like this.

Implement a `/mob/living/simple_animal/proc/wizard_change_name(mob/M)` and use the verbs to call that.